src/main/resources/assets/jei/lang/de_de.lang

```diff
@@ -1,45 +1,45 @@
 # Tooltips
 jei.tooltip.config=JEI-Konfiguration
-jei.tooltip.show.recipes=Zeige Rezepte
+jei.tooltip.show.recipes=Zeige die Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
-jei.tooltip.delete.item=Klicke zum Löschen
+jei.tooltip.delete.item=Klicke, um zu Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
-jei.tooltip.transfer=Bewege Items
+jei.tooltip.transfer=Bewege die Items
```

Comment:
This should stay shortened with the article removed.
```suggestion
jei.tooltip.transfer=Bewege Items
```

Reply:
Is an article not better in this case?

Reply:
Not in concise tooltips.

Reply:
Ok